Gems/ONNX/Code/Source/Clients/Mnist.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "Model.h"
+
+namespace ONNX
+{
+    //! Holds the digit that was inferenced and the time taken for a single inference run.
+    //! Only used by MnistExample().
+    struct MnistReturnValues
+    {
+        int64_t m_inference;
+        float m_runtime;
+    };
+
+    //! Extension of ONNX Model used for Mnist example.
+    //! Implements additional functionality useful to have for the example, such as keeping hold of the input and output vectors, and result
+    //! (which the model doesn't do).
+    struct MNIST
+        : public Model
+        , public AZ::TickBus::Handler
```

Comment:
We can remove the tickbus handler here as it is being executed as part of the tests project.

Reply:
Removed